project1/css/index.css

```diff
@@ -3,62 +3,244 @@
    License: none (public domain)
 */
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+
+   v2.0 | 20110126
+
+   License: none (public domain)
+
+*/
+
+
+
 html, body, div, span, applet, object, iframe,
+
 h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+
 a, abbr, acronym, address, big, cite, code,
+
 del, dfn, em, img, ins, kbd, q, s, samp,
+
 small, strike, strong, sub, sup, tt, var,
+
 b, u, i, center,
+
 dl, dt, dd, ol, ul, li,
+
 fieldset, form, label, legend,
+
 table, caption, tbody, tfoot, thead, tr, th, td,
+
 article, aside, canvas, details, embed, 
+
 figure, figcaption, footer, header, hgroup, 
+
 menu, nav, output, ruby, section, summary,
+
 time, mark, audio, video {
+
 	margin: 0;
-	padding: 0;
-	border: 0;
+
+	padding: 0px;
+
+	border: 0px;
+
 	font-size: 100%;
+
 	font: inherit;
+
 	vertical-align: baseline;
+
+	
+
 }
+
 /* HTML5 display-role reset for older browsers */
+
 article, aside, details, figcaption, figure, 
+
 footer, header, hgroup, menu, nav, section {
-	display: block;
+
+	display:block;
+
 }
+
 body {
+
 	line-height: 1;
+ 
 }
 ol, ul {
 	list-style: none;
 }
+	
+
+
 blockquote, q {
+
 	quotes: none;
+
 }
+
 blockquote:before, blockquote:after,
+
 q:before, q:after {
+
 	content: '';
+
 	content: none;
+
 }
+
 table {
+
 	border-collapse: collapse;
-	border-spacing: 0;
+
+	border-spacing: 0px;
+
 }
 
+
+
+
 /* Set every element's box-sizing to border-box */
+
 * {
+
     box-sizing: border-box;
+
 }
 
+	
+
 html, body {
+
     height: 100%;
+
     font-family: 'Titillium Web', sans-serif;
+
 }
 
+
+
 h1, h2, h3, h4, h5 {
+
     font-family: 'Bangers', cursive;
+
     letter-spacing: 1px;
+
     margin-bottom: 10px;
+    
 }
+  
+	header {
+		display:flex;
+		flex-flow:wrap-reversed;
```

Comment:
It looks like you were trying to reverse the order of the items in the header here, but a couple minor typos tripped you up. Make sure you leave a space between the name of the CSS property and the value (like `display: flex`), and remember that it's row/column-reverse (no 'd' at the end).